gax-httpjson/src/main/java/com/google/api/gax/httpjson/ApiMethodDescriptor.java

```diff
@@ -131,14 +131,86 @@ public ResponseT read(JsonReader in) throws IOException {
   }
 
   ResponseT parseResponse(Reader input) {
-    return responseMarshaller().fromJson(input, responseType());
+    return getResponseMarshaller().fromJson(input, getResponseType());
   }
 
   void writeRequest(Appendable output, RequestT request) {
-    this.requestMarshaller().toJson(request, output);
+    this.getRequestMarshaller().toJson(request, output);
   }
 
   void writeRequestBody(RequestT apiMessage, Appendable output) {
-    httpRequestBuilder().writeRequestBody(apiMessage, requestMarshaller(), output);
+    getHttpRequestBuilder().writeRequestBody(apiMessage, getRequestMarshaller(), output);
+  }
+
+  public static <RequestT, ResponseT> Builder<RequestT, ResponseT> newBuilder() {
+    return new Builder<>();
+  }
+
+  public final Builder<RequestT, ResponseT> toBuilder() {
+    return new Builder<>();
+  }
```

Comment:
Maybe AutoValue can generate this, I'm not sure. Anyway, this should populate a builder with all of the values in the class, as opposed to returning an empty builder.

Reply:
Oh, that was a lingering artifact of copypasta. Removed.